components/ProfileHeader/ProfileHeader.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div class="profile-header">
+    <div class="profile-banner">
+      <img src="https://via.placeholder.com/885x120" alt="profile banner" />
+    </div>
+    <div class="profile-info-container">
+      <div class="profile-info">
+        <div class="profile-info--header">
+          <div class="profile-info--user-activity md:hidden">
+            <p><span class="font-semibold">Runs: </span>{{ runCount }}</p>
+            <p><span class="font-semibold">Games: </span>{{ gameCount }}</p>
+          </div>
+          <div class="profile-info--picture-container">
+            <img src="https://via.placeholder.com/120" alt="profile picture" />
+          </div>
+          <div class="profile-info--user-badges md:hidden">
+            <CoreUserBadges :badges="badges" />
+          </div>
+        </div>
+        <div class="profile-info--user-bio">
+          <h1 class="text-2xl font-semibold">{{ username }}</h1>
+          <h2 class="text-sm text-black text-opacity-50">{{ location }}</h2>
+          <p class="text-black text-opacity-50 pt-2 md:hidden">{{ bio }}</p>
+        </div>
+        <div class="profile-info--follow">
+          <CoreFollowButton />
+        </div>
+        <div class="profile-info--user-activity hidden md:grid">
```

Comment:
this is repeated stuff compared to what is above. Could you make a small component for that part that would only be used in the profileHeader for better visibility and avoid repetition?

Reply:
Yeah, I can break that out for sure.

Reply:
Addressed that with this: https://github.com/speedrun-website/speedrun.website/pull/34/commits/6a726581feaa8c3a4721564035778df65bae3b5a